.github/scripts/publish_post_check.sh

```diff
@@ -0,0 +1,105 @@
+
```

Comment:
This file no longer exists in source. Can be dropped from this PR.

Reply:
Ah, that's what caused this bad merge. (I mean, I should've caught it anyways, but this is making more sense now.)